src/benchmarks/src/include/bench_utils.hpp

```diff
@@ -0,0 +1,71 @@
+#include <unordered_map>
+
+namespace bench_utils {
+
+/*
+ * Keys for config
+ */
+const std::string CIFAR_DATA_PATH = "cifar_data_path";
+const std::string NUM_THREADS = "num_threads";
+const std::string NUM_BATCHES = "num_batches";
```

Comment:
Again, I think it makes sense to define benchmark-specific keys within individual implementation files.